src/components/main/HomeScreen.js

```diff
@@ -215,8 +212,10 @@ export default class HomeScreen extends React.Component {
                 </View>
               </View>
             </View>
-            <MenuBar navigation={this.props.navigation} />
           </ImageBackground>
+          <MenuBar navigation={this.props.navigation} />      
+            <MenuBar navigation={this.props.navigation} />
+          <MenuBar navigation={this.props.navigation} />      
```

Comment:
bruh what. You added 3?

Reply:
lol. idk why that happened i must've accidentally undo'd something before pushing